app/views/organizations/show.html.erb

```diff
@@ -0,0 +1,53 @@
+<p>
+  Hello, <%= @org.name %>
+</p>
+
+<p>
+<% if @organization_campaign %>
+  Your current campaign is <%= @organization_campaign.campaign.name %>
+
+  <table>
```

Comment:
For bootstrap, we'll want to make this `<table class="table">` to style is properly. Because of some other issues going on with the styles this merely stretches it out across the page, so leaving this for now. We should wrap the entire  page layout, minus the navbar & footer, in a `<div class="container">` to constraint it and give it some default edge padding

This is fine for now though